docs/Quickstart.mdx

```diff
@@ -137,17 +160,32 @@ Upon doing so, the response in the Save Pipeline Cell should say "Commit complet
 
 ### Step 6: Run the flow
 
-Deployment takes roughly 1-2 minutes.  Once the deployment is completed, return to the <div className="button darkblue_button"><BranchesOutlined /> **Flow Editor**</div> view and click <div className="button darkblue_button_mui"><ExpandLessOutlined className="white_icon" />**Expand**</div> in the Run Queue bar at the bottom of the screen to expand the input form for running nodes.  Select local storage and the PlateReader.xlsx file downloaded at the beginning of this quickstart.
+Deployment takes roughly 1-2 minutes.  Once the deployment is completed, return to the <div className="button darkblue_button"><BranchesOutlined /> **Flow Editor**</div> view and click <div className="button darkblue_button_mui"><ExpandLessOutlined className="white_icon" />**Expand**</div> in the Run Queue bar at the bottom of the screen to expand the input form for running nodes. Click the "Drag and drop files here or Browse" button and upload the "PlateReader.xlsx" file downloaded at the beginning of this quickstart.
```

Comment:
We should add a note here that the notification page can be used to track the progress of a Save, and a toast message will pop up in the UI when the Save is complete.

Reply:
sg will do